pom.xml

```diff
@@ -1,27 +1,84 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.opentestsystem.common</groupId>
     <artifactId>tds-common-parent</artifactId>
     <name>tds-common-parent</name>
     <packaging>pom</packaging>
     <version>0.0.1-SNAPSHOT</version>
 
+    <scm>
+        <connection>scm:git:https://github.com/SmarterApp/TDS_Common.git</connection>
+        <developerConnection>scm:git:git@github.com:SmarterApp/TDS_Common.git</developerConnection>
+        <url>https://github.com/SmarterApp/TDS_Common</url>
+        <tag>HEAD</tag>
+    </scm>
```

Comment:
This is needed so the 'release:prepare' goal can push version bumps back into the release branch.